Readme.md

```diff
@@ -1,43 +1,52 @@
-**The Readme is to detail the behaviour expected from the Stochiometry widget**
 
-1. When initializing the records of the experiment
+The Readme is to detail the data entry process and the behaviour expected from the Stochiometry widget for LabArchives eNotebook. 
```

Comment:
I think this should start with a heading, as it did in a previous comment of mine.